pgm-2.txt

```diff
@@ -0,0 +1,7 @@
+
+#!/bin/bash
+echo "hellooooooo"
+find *.log > z.txt
+find *.txt >> z.txt
+sudo mkdir /home/sudhir/zout
+tar -zczf matches.tar.gz /home/main
```

Comment:
never mind for the tar and then compress, it seems that the -z switch accomplishes this effect.